src/index.ts

```diff
@@ -63,7 +74,13 @@ async function webtransportBiDiStreamToStream (bidiStream: any, streamId: string
   let writerClosed = false
   let readerClosed = false;
   (async function () {
```

Comment:
Could lose the IIFE and just wrap the contents in a try/catch?

Reply:
I don't think so? `await writer.closed` will block until the writer is closed. We could get rid of the iife by doing something like:

```
writer.closed.catch(err => err).then(maybeErr => { ...
```

Do you think that's better?